tests/benchmark_test.py

```diff
@@ -195,46 +215,25 @@ def test_create_person_class_from_data(mock_collect_user_input, temp_testbed):
         " Note that the father is listed"
         " first in a parents list."
     )
-    # makes linter shut up
-    easy_prompt
-    easy_medium_prompt
-    medium_prompt
-    hard_prompt
     mock_collect_user_input.side_effect = [
         easy_medium_prompt,
         "y",
         KeyboardInterrupt,
     ]
     run(["person_data/data.json", "person_data/person.py"])
 
-    # This imports the Person class from the file mentat edited just created
-    # import in Python is always relative to the file's original path
-    # so even though we change the cwd, we can't use relative imports to import form the tmp dir
-    file_path = os.path.join(os.getcwd(), "person_data/person.py")
-    module_name = "testbed.person_data.person"
-    spec = importlib.util.spec_from_file_location(module_name, file_path)
-    module = importlib.util.module_from_spec(spec)
-    sys.modules[module_name] = module
-    spec.loader.exec_module(module)
-    Person = module.Person
-    # from testbed.person_data.person import Person
+    relative_file_path = "person_data/person.py"
+    Person = import_module_from_path("", relative_file_path).Person
+    # from testbed.person_data.solution_person import Person
```

Comment:
This is more like writing from testbed import person_data.person; Person = person_data.Person